resteasy-upgrade-guide/reference/en/en-US/modules/Interceptors.xml

```diff
@@ -8,7 +8,6 @@
    
    <orderedlist>
       <listitem>reader/writer interceptors</listitem>
-      <listitem>server side <classname>PreProcessInterceptor</classname></listitem>

```

Comment:
The changes to PreProcessInterceptor (deprecated in 3.x and removed in 4.x) applies to the whole RESTEasy 2 interceptor mechanism. So this whole Interceptors.xml chapter has to be revisited.

Reply:
All references to PreProcessInterceptor are already removed for the book.